docs/api/ethereum.md

```diff
@@ -1030,6 +1045,256 @@ A class instance `variable` holding a `string` which represents smart contract a
 
 A class instance `variable` holding a `string` which represents the URL to the compiled ERC-20 related smart contract definition file. This file is used when deploying new value ledgers to the network.
 
+## Deploy gateway
+
+Deploy gateway allows for delegating deployment of an `AssetLedger` smart contract to a third party and transfering value in the same atomic transaction.
```

Comment:
Deploy gateway allows for delegating deployment of an `AssetLedger` smart contract to a third party and for transferring value in the same atomic transaction.